model/jpa/src/main/resources/META-INF/jpa-changelog-master.xml

```diff
@@ -69,5 +69,6 @@
     <include file="META-INF/jpa-changelog-11.0.0.xml"/>
     <include file="META-INF/jpa-changelog-12.0.0.xml"/>
     <include file="META-INF/jpa-changelog-13.0.0.xml"/>
+    <include file="META-INF/jpa-changelog-14.0.0.xml"/>
```

Comment:
By recent keycloak's convention, it might be better not to add or modify JPA entity because it causes a schema change. For example, Client Policies/Profiles avoid adding entities by storing JSON string itself to realm's attribute (existing key-value store).

Reply:
will be remove and stored as described above